src/website/app/Label.js

```diff
@@ -20,28 +20,29 @@ import { createStyledComponent } from '../../utils';
 
```

Comment:
I think Badge (or something else) is a better name than Label for this component. Only because we’ll likely promote it to the system at some point, and this could be confused with labels that go with form inputs.

Reply:
i'm not going to change the name of this component for this PR, we can do that when we promote it though.